pipeline/export.py

```diff
@@ -460,44 +461,41 @@ def _export_recording(insert_key, output_dir='./', filename=None, overwrite=Fals
     print('ok.')
 
 
-def write_to_activity_viewer_json(probe_insertion, filepath=None, per_period=False):
-    probe_insertion = probe_insertion.proj()
-    key = (probe_insertion * lab.WaterRestriction * experiment.Session).proj('session_date', 'water_restriction_number').fetch1()
-    uid = f'{key["subject_id"]}({key["water_restriction_number"]})/{datetime.strftime(key["session_date"], "%m-%d-%Y")}({key["session"]})/{key["insertion_number"]}'
+def write_to_activity_viewer(insert_keys, output_dir='./'):
+    """
+    :param insert_keys: list of dict, for multiple ProbeInsertion keys
+    :param output_dir: directory to write the npz files
+    """
 
-    units = (ephys.UnitStat * ephys.Unit * lab.ElectrodeConfig.Electrode
-             * histology.ElectrodeCCFPosition.ElectrodePosition
-             & probe_insertion & 'unit_quality != "all"').fetch(
-        'unit', 'ccf_x', 'ccf_y', 'ccf_z', 'avg_firing_rate', order_by='unit')
+    if not isinstance(insert_keys, list):
+        insert_keys = [insert_keys]
 
-    if len(units[0]) == 0:
-        print('The units in the specified ProbeInsertion do not have CCF data yet')
-        return
+    for key in insert_keys:
+        water_res_num, sess_date = get_wr_sessdate(key)
 
-    penetration_group = {'id': uid, 'points': []}
+        uid = f'{water_res_num}_{sess_date}_{key["insertion_number"]}'
```

Comment:
does this need to handle multi-session/day?

Reply:
it currently does, `sess_date` there is actually `sess_datetime`